Gems/AudioSystem/Code/Source/Editor/AudioControlsEditorWindow.h

```diff
@@ -31,7 +34,6 @@ namespace AudioControls
     //-------------------------------------------------------------------------------------------//
     class CAudioControlsEditorWindow
         : public QMainWindow
-        , public Ui::MainWindow
```

Comment:
Breaks in Qt6 and its one of the only occurence where we inherit UI file instead of having it as a member. Moving it as a member fixes the compilation and widget behaves the same